index.js

```diff
@@ -65,7 +65,7 @@ class SnackbarComponent extends Component {
           <Text style={[styles.text_msg, { color: this.props.messageColor }]}>
             {this.props.textMessage}
           </Text>
-          {this.props.actionHandler && this.props.actionText && (
+          {this.props.actionHandler && Boolean(this.props.actionText) && (
```

Comment:
This would not necessarily fix it because at default javascript booleans every condition parameter so `this.props.actionsText` and `Boolean(this.props.actionText)` is no different, but you can set the props to required if thats the case and that would be better

Reply:
I've deployed this fix to production, so it does work.

---

Before #28 this was `{undefined && undefined && <Comp>}`, but #28 changed it to `{(() => {}) && '' && <Comp>}`.

```
console.log(undefined && undefined && 'render');
undefined
console.log((() => {}) && '' && 'render');
''
```

The real fix is probably to remove the default props added in #28 since we don't need any default values for them. But whatever, do what you want 🤷‍♂

Reply:
`console.log(''? 'a' : 'b') // b`

The real props that makes the condition to pass is the `this.props.actionHandler` because that has a default value of function

Reply:
Its best if you look at #17 which has a better fix

Reply:
#17 was merged with the exact same change as this PR except it does `!!` instead of `Boolean`

Reply:
Its is not merged yet however it does check for the `this.props.actionHandler` before using the boolean which is still not necessary

Reply:
What are you talking about?

![image](https://user-images.githubusercontent.com/1404810/66906502-1e48c180-f008-11e9-8f90-fc1a80754ff1.png)

With that I'm out

Reply:
I guess it was only suggested from the comment but wasn't implemented afterwards